crates/typst-library/src/meta/numbering.rs

```diff
@@ -102,7 +102,19 @@ impl Numbering {
 
     /// Create a new `PdfNumbering` from a `Numbering` applied to a page
     /// number.
-    pub fn apply_pdf(&self, number: usize) -> PdfPageLabel {
+    pub fn apply_pdf(&self, vt: &mut Vt, number: usize) -> PdfPageLabel {
+        // If the numbering function returns something that can reasonably be made into text,
+        // we use that as the logical page prefix.
+        if let Numbering::Func(func) = self {
+            // TODO: needs at least two arguments in `args`, don't know what to put as the second one.
```

Comment:
The function will always want both the current and total page. It's a bit problematic since here we want its output for just the page and the function might not be prepared for that.

Reply:
What do you mean "might not be prepared for that"? But yea, we don't have access to the total page number here, that's true.

Reply:
I mean that PDF wants just the page nunber itself and not it and the total (right?). But if `page.numbering` is set to a function, the user passes a function that might or might not show both, we don't know. If they did write a function that shows both, there is no way to get the page number without the total for PDF.

Reply:
The PDF does just want the current page number, correct. I mean, currently this implementation more or less ignores what the user specifies in the function and just uses `number` from the `layout` signature.